compiler/rustc_data_structures/src/sorted_map.rs

```diff
@@ -175,19 +175,19 @@ impl<K: Ord, V> SortedMap<K, V> {
 
         let drain = match start_index {
             Ok(index) => {
-                let mut drain = elements.drain(..);
+                let mut drain = elements.into_iter();
```

Comment:
the variable name isn't appropriate anymore after the change

Reply:
Done